src/www/system_general.php

```diff
@@ -348,6 +356,21 @@
                 </div>
               </td>
             </tr>
+            <tr>
+              <td><a id="help_for_navigation" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Navigation"); ?></td>
+                <td>
+                  <input name="navigation" class="navigation" id="navigation_standard" type="radio" value="standard" <?= $pconfig['navigation'] == "standard" ? 'checked="checked"' : '' ?>/>
+                  <?=gettext("Standard"); ?>
+                  &nbsp;&nbsp;&nbsp;
```

Comment:
try to avoid non breaking spaces - it is an ugly hack from old days. you could use inline blocks etc. which have a padding and margin or switch to a select box